frontend/templates/NOTES.txt

```diff
@@ -49,9 +48,16 @@ SSH connection (limited access through VPN):
 
   {{ range $index, $service := .Values.services -}}
   {{- if $service.mounts }}
-  {{- range $indexMount, $service.mounts -}}
-  Importing files to service '{{ $index }}', folder '{{ (index $.Values.mounts $indexMount).mountPath }}':
-rsync -azv -e 'ssh -A -J {{ include "frontend.jumphost" $ }}' local_folder/ www-admin@{{ template "frontend.environment.hostname" $ }}-shell-{{ $index }}.{{ $.Release.Namespace }}:{{ (index $.Values.mounts $indexMount).mountPath }}
+File mounts ('{{ $index }}' service):
+  {{- range $indexMount, $service.mounts }}
+  {{ $folder := (index $.Values.mounts $indexMount).mountPath }}
```

Comment:
Can we have `{{-` instead of `{{` here? It's not very important, but it will leave empty lines in output.

Reply:
Done